### ISSUE TITLE
Feature Request: Word Search Box (Function) 

### ISSUE THREAD

**Issue by strable** [learnweb/moodle-mod_moodleoverflow#67]
We are using MoodleStack to answer questions about English for our engineering team, and it would be useful to have the ability to search for specific words as the original StackOverflow does. The database can become large, and a search function helps find the data. 

Many thanks for considering this request. It is great for me to find MoodleStack. The plug-in fits our purposes of creating an English Stack in Moodle just as the developers do with the original StackOverflow.

**Comment by amromohd** [comment]
I hope this function will be implemented soon because it's so important to my students.

**Comment by jeroenkeppens** [comment]
I agree that such a feature would greatly improve usability of moodle overflow at scale.

**Comment by NinaHerrmann** [comment]
Hey together, 
we are working on inserting moodleoverflow into global search (https://github.com/learnweb/moodle-mod_moodleoverflow/pull/62).
Feel free to check out that branch to test the functionality, as we currently do not find the time to review it.

**Comment by strable** [comment]
Thank you!
